source/js/fellowships.js

```diff
@@ -0,0 +1,183 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "React" }] */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Person from './components/people/person.jsx';
+import LoadingIndicator from './components/loading-indicator/loading-indicator.jsx';
+
+let pulseApiDomain = ``;
+let pulseDomain = ``;
+const DIRECOTRY_PAGE_FILTER_OPTIONS = {'program_year': `2017`};
+const DIRECTORY_PAGE_TYPE_ORDER = [ `science`, `open web`, `tech policy`, `media`];
+
+function getFellows(params, callback) {
+  Object.assign(params, {'profile_type': `fellow`});
+
+  let queryString = Object.entries(params).map(pair => pair.map(encodeURIComponent).join(`=`)).join(`&`);
+  let req = new XMLHttpRequest();
+
+  req.addEventListener(`load`, () => {
```

Comment:
This could be a one-liner:

`req.addEventListener(`load`, () => callback.call(this, JSON.parse(req.response)));`

Do we really need to bind the callback to `this`?